### DISCUSSION TITLE
How to get the LaTeX typewriter font?

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#2296] [Q&A] [answered]
I have tried 

```
#set text(font: "New Computer Modern")
#show raw: set text(font: "New Computer Modern Mono")
```

but with no success.

I am trying to obtain:

![image](https://github.com/typst/typst/assets/75945439/53a9628a-5088-4eab-9b4e-ae53d4401ab0)

Thanks!

**Comment by ghost** [comment, marked as answer]
Install [regular](https://ftp.agdsn.de/pub/mirrors/latex/dante/fonts/newcomputermodern/otf/NewCMMono10-Regular.otf) and [italic](https://ftp.rrze.uni-erlangen.de/ctan/fonts/newcomputermodern/otf/NewCMMono10-Italic.otf) shape.
Then it should work:

```typst
#set text(font: "New Computer Modern", size: 11pt)
#show raw: set text(font: "New Computer Modern Mono", size: 11pt)

Normal font and `typewriter` and _`typewriter`_ font
```

![typewriter](https://github.com/typst/typst/assets/138529922/e810e89d-e6fe-4af4-a607-323b303c16f0)

**Comment by psads-git** [comment]
Thanks, @gfpj! Where should the fonts be placed?

**Reply by ghost** [reply]
@psads-git As far as I know, in Typst you have to install them. In Windows, this is done by double clicking on the font files, in Linux there is a default font folder. Typst searches for the fonts in a specific folder, you can list all the fonts found by running `typst fonts`.

**Reply by psads-git** [reply]
Thanks, @gfpj, for your help!
